Assets/Python/Screens/CvTechChooser.py

```diff
@@ -870,7 +869,8 @@ def placeGreatPeople(self):
 						iMaxFlavor = gc.getTechInfo(iLoopTech).getFlavorValue(iFlavor)
 						iTech = iLoopTech
 
-			screen.addDDSGFCAt("GreatPeopleUnit" + str(iUnit),"TechBottomPanel", gc.getUnitInfo(iUnit).getButton(), iGPX, 16, self.GP_ICON_SIZE, self.GP_ICON_SIZE, WidgetTypes.WIDGET_PEDIA_JUMP_TO_UNIT, iUnit, -1, False)

```

Comment:
Don't we want to keep the ability to jump to the GP entry when clicking the button?

Reply:
That was also one of my concerns. I replaced that because the old tech screen didn't have a pedia jump at all. I'll try to come up with a solution. (I actually also prefer to have the pedia jump available)

Reply:
Okay good. It looks like it should work as it was though, what is the issue there?